pipeline/transform/sidewalk_cafe.py

```diff
@@ -0,0 +1,76 @@
+import sys
+sys.path.append("./")
+
+import argparse
+import pandas as pd
+import numpy as np
+from timeit import default_timer as timer
+from datetime import date
+from utils.data import extract_data_portal_dates
+from utils.geo import convert_zip_to_community_area_proportionally
+
+cli = argparse.ArgumentParser(description="Transform sidewalk cafe permit data.")
+cli.add_argument("--zip_to_area_file", help="File to transform zip to community area.")
+cli.add_argument("--area_to_name", help="File to transform area number to community name.")
+cli.add_argument("--input_file", help="File path to read sidewalk cafe permit data from.")
+cli.add_argument("--output_file", help="File path to write results to.")
+args = cli.parse_args()
+
+start = timer()
+
+
+#Read data
+raw_df = pd.read_csv(args.input_file)
+
+
+#Parse dates
+df = extract_data_portal_dates(raw_df, col="issued_date")
+df = extract_data_portal_dates(df, col="expiration_date")
+
+#Drop lat lon columns (too many null)
+df.drop([
+    "latitude",
+    "longitude",
+], axis=1, inplace=True)
+
+
+#Create new column "exists". Assign 0 for expired permits, 1 for existing permits
+today = date.today().strftime("%Y-%m-%d")
+df['exists'] = np.where( (df['issued_date'] < today ) & (today < df['expiration_date']), 1, 0)
```

Comment:
I would recommend that we check whether the permit is existing or expired in the API rather than in the offline pipeline, that way we can use the current date instead of the date when we ran our pipeline.

Or was there another reason for adding the exists column in the offline pipeline?